UPGRADE.md

```diff
@@ -0,0 +1,37 @@
+# Upgrade guide to Smooch
+
+SupportKit is now Smooch!
+
+To reflect this transition, we bumped up the major version to 2.0. We introduced a lot of breaking changes in the internals of the library to let it hit our brand new public API.
```

Comment:
Announcing breaking changes sounds harsh and might discourage people from adopting it. In reality, none of the SDK methods have changed, so moving to smooch should be easy for most integrators. Avoiding the term breaking changes might communicate this better. EG:

```With Smooch, the internals of the SDK have changed. It is now using our public API (<--make this a hotlink) and as such we've bumped the major version to 2.0```